lib/ex_force.ex

```diff
@@ -236,6 +250,10 @@ defmodule ExForce do
   def query_all_stream(soql, config \\ default_config()),
     do: start_query_stream(&query_all/2, soql, config)
 
+  defp extract_create_id(resp = %{"id" => id, "success" => "true"}) do
```

Comment:
Let's make it inline in caller side, like this:

```elixir
{201, %{"id" => id, "success" => true}} -> {:ok, id}
```

Reasons & Note:

- The extraction implementation is too simple to be in a separate function - just plain pattern match would work
- Do all create requests return responses in this structure? Is this structure only used by create? I'd defer making abstraction..

Reply:
Excuse my newbie experience in Elixir :) been only writting it for a month

As far as I can tell only the create seems to have that structure. And even more bizzarre, the success is never sent on error. nor the errors either. they come in a different format. So I am not seeing an documentation on that either beside that one page